src/app/screens/Send.tsx

```diff
@@ -35,8 +37,11 @@ function Send() {
       }
 
       if (lnurl) {
-        await lnurlLib.getDetails(lnurl); // throws if invalid.
-        navigate(`/lnurlPay?lnurl=${lnurl}`);
+        navigate(URLS.lnurlPay, {
+          state: {
```

Comment:
shouldn't we say that we always pass the lnurlpay details to the state?

as you also removed the `getDetails` here the error case is no longer handled.

Reply:
The component `LNURLPay` does exactly that check [here](https://github.com/getAlby/lightning-browser-extension/pull/1257/files#diff-0ae7727e94b1752f52e903368c32a80af0ae7d22e7d4d4e17bb6b7219fe7a076R37)

Therefore one can navigate to the view with passing a "lnurl"  only in the navigation.state.
The component does the rest.

Reply:
@lisabaut yes, but I don't get the point of that we already load the lnurldetails here (and we must do that because we need to check if it is a valid LNURL and especially what kind `lnurlDetails.tag`) of lnurl this is to navigate to the correct route (lnurlpay, lnurlauth, etc.)

so why keep that complexity in the LNURLPay component?

Reply:
Sure, the component would not take care of the other views.
This PR is a first version of showing how to use the navigation state with the the view "LNURPay" as a start for further views.

To sum up:

- the check if the lnurl is valid should be done in components HOME and SEND
- `lnurlLib.getDetails` should get missing details in components HOME and SEND 
- missing origin data should be created in components HOME and SEND
- `origin` & `args.lnurDetails` should be attached to the navigation.state when navigating to LNURLPay from HOME and SEND
- the wrapper component for the actual LNURLPay view should be removed
- the LNURLPay view should read the navigationState directly

Reply:
now I am confused, why do we have it then if it should be removed again? (we already touch the lnurlpay component, don't we?)

Reply:
Now I am confused too 😄 

The sum-up above is the way you want to change the code to remove "the complexity" in LNURLPay, right?